web/scripts/generate-llm-files.ts

```diff
@@ -1,92 +1,104 @@
-const LLMS_FULL_TXT_FILE_PATH = path.join(STATIC_DIR, LLM_FULL_FILENAME)
-const CATEGORIES_TO_IGNORE = ['Miscellaneous']
+  "https://raw.githubusercontent.com/wasp-lang/wasp/refs/heads/release/web/"; // Use the release branch
+const WASP_BASE_URL = "https://wasp.sh/";
+const LLM_FULL_FILENAME = "llms-full.txt";
+const LLM_OVERVIEW_FILENAME = "llms.txt";
+const LLMS_TXT_FILE_PATH = path.join(STATIC_DIR, LLM_OVERVIEW_FILENAME);
+const LLMS_FULL_TXT_FILE_PATH = path.join(STATIC_DIR, LLM_FULL_FILENAME);
+const CATEGORIES_TO_IGNORE = ["Miscellaneous"];
 
 const OVERVIEW_INTRO_CONTENT = `
 # Wasp
 Wasp is a full-stack framework with batteries included for React, Node.js, and Prisma.
 
 ## Individual documentation sections and guides:
-`
+`;
 const OVERVIEW_MISC_SECTION_CONTENT = `
 ## Miscellaneous
 - [Wasp Developer Discord](https://discord.com/invite/rzdnErX)
 - [Open SaaS -- Wasp's free, open-source SaaS boilerplate starter](https://opensaas.sh)
-`
+`;
 
-await generateFiles()
+(async () => await generateFiles())();
 
 /**
  * Main function to generate the LLM-friendly doc files.
  * It orchestrates the process by fetching and processing documentation and blog files,
  * and then writing the final output to the static directory.
  */
 async function generateFiles() {
-  console.log('Starting LLM file generation...')
+  console.log("Starting LLM file generation...");
+
+  if (!Array.isArray(docsSidebarConfig.docs)) {
+    throw new Error(
+      "Sidebar configuration for docs is not an array. Please check the sidebars.ts file.",
+    );
+  }
 
   const { overviewDocsSection, fullConcatContent } =
-    await processDocumentationFiles(docsSidebarConfig.docs)
-  const blogSectionContent = await processBlogFiles()
+    await processDocumentationFiles(docsSidebarConfig.docs);
+  const blogSectionContent = await processBlogFiles();
 
   const llmsTxtContent =
     OVERVIEW_INTRO_CONTENT +
     overviewDocsSection +
     blogSectionContent +
-    OVERVIEW_MISC_SECTION_CONTENT
-  const llmsFullTxtContent = fullConcatContent
+    OVERVIEW_MISC_SECTION_CONTENT;
+  const llmsFullTxtContent = fullConcatContent;
 
-  await writeOutputFiles(llmsTxtContent, llmsFullTxtContent)
-  console.log('🎉 LLM file generation complete.')
+  await writeOutputFiles(llmsTxtContent, llmsFullTxtContent);
+  console.log("🎉 LLM file generation complete.");
 }
 
 /**
  * Processes all documentation files based on the sidebar configuration's order.
  * It builds a map of document IDs to file paths, processes each document,
  * and generates both a summarized overview and a full concatenated string of the content.
- * @param {object[]} sidebarConfig - The `docs` array from the sidebars.js configuration.
+ * @param {object[]} sidebarConfigDocs - The `docs` array from the sidebars.js configuration.
  * @returns {{overviewDocsSection: string, fullConcatContent: string}} - An object containing the generated overview section and the full concatenated content.
```

Comment:
We can remove the JSDoc typings throughout the file, TS will ignore them in ts files